### PR TITLE
[BRAN-151] Added Nunito as font

### PR DESCRIPTION
**Pull request by jeffarrubla**
## Purpose/Description:

- Added Nunito as font family by default.
- Added dependecy from npm for Nunito font.

## Screenshots:

image
## Jira Link:

[BRAN-151 Add Nunito Sans Font - Mosaic](https://collagegroup.atlassian.net/browse/BRAN-151)

## Dependencies:

No dependencies

## Test Plan:

-   **Test Scenario 1:** Run the project

    -   **Steps:**
        1. Select Typography
        2. Pick, default, header or body
        3. Use developer tools, pick inspector tab.
        4. Select text for default: Default, for header: Heading and body: Body
    -   **Expected Result:** In the css you should see `font-family: 'Nunito',sans-serif;` (as you can see in the attachment)
    
### Reviewer Instructions:

None

### Deployment Considerations:

None

### CODE REVIEW

**Review by Alderr — CHANGES_REQUESTED** [review]
Almost there, you need to add MuiCSSBaseline so that we can use the font for everything in the dashboard. Then you have to inform MUI Typography about the fontWeights: 500, 700, 900. Make sure to look at MUI `v5` solutions on the internet. not `v4`

https://mui.com/material-ui/customization/typography/

File: src/theme/theme.ts [hunk 1]
```diff
@@ -99,6 +103,209 @@ const themeOptions: ThemeOptions = {
...
+        font-family: 'Nunito Sans';
+        font-style: normal;
+        font-weight: 900;
+        font-stretch: 100%;
+        font-display: swap;
+        src: local('Nunito Sans') format('woff2');
+        unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+      }
+      `,
+    },
```
Comment (Alderr) [inline src/theme/theme.ts]: Can you add stories to the `Typography` component that use the `fontWeight` prop? 

```javascript
<Typography variant="b2" sx={{ fontWeight: 700 }} />
```

File: src/theme/theme.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import '@fontsource/nunito-sans';
```
Comment (Alderr) [inline src/theme/theme.ts]: if you already imported `nunito-sans` through CSS, do you need this?

File: package.json [hunk 3]
```diff
@@ -114,5 +115,8 @@
             "last 1 firefox version",
             "last 1 safari version"
         ]
+    },
+    "dependencies": {
```
Comment (Alderr) [inline package.json]: same here, if you imported `nunito-sans` through CSS, do you need this?

File: package-lock.json
Comment (Alderr) [inline package-lock.json:1]: why is your npm adding 6,000 lines to the `package-lock.json`? LOL

File: src/theme/theme.ts [hunk 4]
```diff
@@ -99,6 +103,209 @@ const themeOptions: ThemeOptions = {
         },
       },
     },
+    MuiCssBaseline: {
+      styleOverrides: `/* cyrillic-ext */
+      @font-face {
```
Comment (diego20) [inline src/theme/theme.ts]: I am lost here. Is it all necessary? What these do? What happens if these are not added to the code?
Material's docs seem shorter but I don't know in all certainty. https://mui.com/material-ui/customization/typography/#font-family

File: src/theme/theme.ts [hunk 5]
```diff
@@ -99,6 +103,209 @@ const themeOptions: ThemeOptions = {
...
+        font-family: 'Nunito Sans';
+        font-style: normal;
+        font-weight: 400;
```
Comment (Alderr) [inline src/theme/theme.ts]: you can use this pattern inside `MuiCssBaseline`: 

```javascript
import AbradeRegular from 'assets/fonts/Abrade-Regular.ttf'
import AbradeMedium from 'assets/fonts/Abrade-Medium.ttf'
....

const abradeRegular = {
  fontFamily: 'Abrade',
  fontStyle: 'normal',
  fontWeight: 'normal',
  src: `url(${AbradeRegular}) format('truetype')`,
}

const abradeMedium = {
  fontFamily: 'Abrade',
  fontStyle: 'normal',
  fontWeight: 500,
  src: `url(${AbradeMedium}) format('truetype')`,
}
....

theme.components = {
  MuiCssBaseline: {
    styleOverrides: {
      html: [
        {'@font-face': abradeLight},
        {'@font-face': abradeRegular},
        {'@font-face': abradeMedium},
        {'@font-face':abradeSemiBold},
        {'@font-face': abradeBold},
        {'@font-face':abradeExtraBold},
        {'@font-face':abradeBlack}
      ],
      'html, body': {
        padding: 0,
        scrollbarWidth: 'thin',
      },
}}}
```

File: src/theme/theme.ts (hunk 4 above)
Comment (Alderr) [reply src/theme/theme.ts]: @diego20 if we want to add a font to mosaic AND our dashboard, we need `MuiCSSBaseline`

**Review by diego20 — APPROVED** [review]

**Review by Alderr — COMMENTED** [review]
added some changes related to font not appearing and I tweaked the typography component as well.

**Comment by jeffarrubla** [comment]
LGTM, I can't approve it

**Review by Alderr — APPROVED** [review]

**Pull request merged**